internal/cephfs/controllerserver.go

```diff
@@ -538,12 +543,13 @@ func cleanUpBackingVolume(
 	volID *store.VolumeIdentifier,
 	cr *util.Credentials,
 	clusterName string,
+	setMetadata bool,
```

Comment:
same here, consider making `cleanUpBackingVolume()` a method of `ControllerServer` to reduce the number of arguments that the function requires